### PR TITLE
Dependend python packages updated

### LINKED ISSUE #1: Fix pyYAML Dependencie (CVE-2017-18342)
Update requirements.txt to use new pyYAML Version.

### PR DESCRIPTION
**Pull request by fager**
fix #1

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,82 +1,84 @@
...
 Send2Trash==1.5.0
 simplegeneric==0.8.1
-six==1.11.0
+six==1.12.0
 smartypants==2.0.1
 terminado==0.8.1
 testpath==0.4.2
 toml==0.10.0
-tornado==5.1.1
+tornado==6.0.1
```
Comment (jokey2k) [inline requirements.txt:74]: tornado 6.0.2 has a fix for
WebSocketHandler.set_nodelay
which may be helpful here

**Review by jokey2k — CHANGES_REQUESTED** [review]
Generally ok, may need this tweak

**Review by jan-lugfl — APPROVED** [review]
looks good to me, without actually testing it

**Comment by jan-lugfl** [comment]
fixed by 933f0cad7b868186114a3fae594e13341101147d

**Pull request closed without merging**